compiler/backend/src/org/jetbrains/kotlin/codegen/state/KotlinTypeMapper.kt

```diff
@@ -615,6 +617,9 @@ class KotlinTypeMapper @JvmOverloads constructor(
             return name
         }
 
+        if (isIrBackend)
```

Comment:
Looks like this ends up duplicating some code. Could we refactor this along these lines:

```
return if (isIrBackend) mangleNonInlineClassMemberNames() else mangleMemberNames()
```

And then have:

```
mangleMemberNames() {
  // all the early returns for inline class members.
  ...
  return mangleNonInlinClassMemberNames()
}
```

The we have sharing of the common parts in `mangleNonInlineClassMemberNames()`?

Reply:
I changed it to get rid of the duplication.